docs/flink-configuration.md

```diff
@@ -148,13 +148,14 @@ INSERT INTO tableName /*+ OPTIONS('upsert-enabled'='true') */
-| compression-codec      | Table write.(fileformat).compression-codec | Overrides this table's compression codec for this write      |
-| compression-level      | Table write.(fileformat).compression-level | Overrides this table's compression level for Parquet and Avro tables for this write |
-| compression-strategy   | Table write.orc.compression-strategy       | Overrides this table's compression strategy for ORC tables for this write |
+| Flink option           | Default                                                              | Description                                                                                                                        |
+|------------------------|----------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------|
+| write-format           | Table write.format.default                                           | File format to use for this write operation; parquet, avro, or orc                                                                 |
+| target-file-size-bytes | As per table property                                                | Overrides this table's write.target-file-size-bytes                                                                                |
+| upsert-enabled         | Table write.upsert.enabled                                           | Overrides this table's write.upsert.enabled                                                                                        |
+| overwrite-enabled      | false                                                                | Overwrite the table's data, overwrite mode shouldn't be enable when configuring to use UPSERT data stream.                         |
+| distribution-mode      | Table write.distribution-mode                                        | Overrides this table's write.distribution-mode                                                                                     |
+| compression-codec      | Table write.(fileformat).compression-codec                           | Overrides this table's compression codec for this write                                                                            |
+| compression-level      | Table write.(fileformat).compression-level                           | Overrides this table's compression level for Parquet and Avro tables for this write                                                |
+| compression-strategy   | Table write.orc.compression-strategy                                 | Overrides this table's compression strategy for ORC tables for this write                                                          |
+| write-parallelism      | The same as the parallel of the input operator to the write operator | Configuring the write parallel number for iceberg stream writer. By default, it is the same as the parallel of the input operator. |
```

Comment:
The same as the parallel of the input operator to the write operator
-->
Upstream operator parallelism

Configuring the write parallel number for iceberg stream writer. By default, it is the same as the parallel of the input operator.
-->
Overrides the writer parallelism

Reply:
Then maybe we can revert the table width change.